.github/workflows/docsbuild-release.yaml

```
@@ -0,0 +1,73 @@
+name: ansible-release-porting-guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible_version:
+        description: 'Release Version. Example : 11.1.0'
+        required: true
+      ansible_major_version:
+        description: 'Exmaple 11'
+        required: true
+
+env:
+  CI_COMMIT_MESSAGE: 'Add Ansible community "${{ inputs.ansible_version }}" porting guide'
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  CI_PR_BODY: |
+
+      ##### SUMMARY
+
+      ##### ISSUE TYPE
+
+      -   Docs Pull Request
+
+      ##### COMPONENT NAME
+
+      docs/docsite/rst/porting_guides/'porting_guide_"${{ inputs.ansible_major_version }}".rst'
```

No need to add double quotes as this is templated before bash. Of did you want to allow the end-users to supply things like $some_var as a version, rather than an actual number?
```suggestion
      docs/docsite/rst/porting_guides/porting_guide_${{ inputs.ansible-major-version }}.rst
```
(though the single quotes probably aren't needed either)

---

Yes, the end user is providing the manual input for `ansible-major-version`.

---

Removed the single quotes as suggested.